docs/developers/v2/hacking-with-yearn.md

```diff
@@ -0,0 +1,237 @@
+# Hacking on Yearn
+
+This page is for the hackers and buidlers who are currently building, or planning to build, something cool with Yearn Finance. This guide  details all the info you need to start hacking!
+
+## Contents
+- Vault and Smart Contract Integrations
+- Front-end and SDK Integrations
+- Strategy Development
+- Data Analysis
+- Inspiration
+- Examples
+- Expectations
```

Comment:
none of the articles have a ToC. I don't think we need it here. You can describe very very briefly what they are going to find in the article in the intro.